website/static/js/projectSettings.js

```diff
@@ -8,77 +8,114 @@ var $osf = require('js/osfHelpers');
 var oop = require('js/oop');
 var ChangeMessageMixin = require('js/changeMessage');
 
-var NodeCategorySettings = oop.extend(
+var ProjectSettings = oop.extend(
     ChangeMessageMixin,
     {
-        constructor: function(category, categories, updateUrl, disabled) {
+        constructor: function(params) {
             this.super.constructor.call(this);
-
             var self = this;
 
-            self.disabled = disabled || false;
-            self.UPDATE_SUCCESS_MESSAGE = 'Category updated successfully';
-            self.UPDATE_ERROR_MESSAGE = 'Error updating category, please try again. If the problem persists, email ' +
+            self.decodedTitle = params.currentTitle;
+            self.decodedDescription = $osf.htmlDecode(params.currentDescription);
+            self.title = ko.observable(params.currentTitle).extend({
+                required: {
+                    params: true,
+                    message: 'Title cannot be blank.'
+                }});
+            self.description = ko.observable(self.decodedDescription);
+
+            self.disabled = params.disabled || false;
+
+            self.UPDATE_SUCCESS_MESSAGE = 'Category, title, and description updated successfully';
+            self.UPDATE_CATEGORY_ERROR_MESSAGE = 'Error updating category, please try again. If the problem persists, email ' +
```

Comment:
@abought Thanks for the input. This error message was created before I got to this issue, so I just assumed it was adequate but you're saying makes sense. 
I'm not sure why a category change might be rejected